mclagsyncd/mclaglink.h

```diff
@@ -204,7 +204,6 @@ namespace swss {
             std::string m_system_mac;
             std::set<vlan_mbr> m_vlan_mbrship; //set of vlan,mbr tuples
 
-            const int MSG_BATCH_SIZE;
```

Comment:
```suggestion
            static const int MSG_BATCH_SIZE = 256;
```

Reply:
I think this fix is easier than defining as a macro.

Reply:
yea, accepted.